core/sqf/export/limited-support-tools/LSO/README

```diff
@@ -152,40 +153,52 @@ URRENT_TIMESTAMP           NODE         PID          EXE_MEM_HIGH_WM_MB  EXE_MEM
---- 1 row(s) selected.
-                                                                                                                                                                     
 QUERIES BLOCKED IN CLIENT
+=========================
 
 This filtering mechanism lists the querie s which are not blocked in SQL longer
-than the given number of seconds and haven't finished execution. The 
-SQL commands to list queries blocked in the client are available at:
+than the given number of seconds and haven't finished execution. 
 
+The SQL commands to list queries blocked in the client are available at:
 $MY_SQROOT/export/limited-support-tools/LSO/queries_in_client.sql      
 
 Example output is below:   
+CURRENT_TIMESTAMP           TIME_IN_SECONDS       QUERY_ID                                                                      EXECUTE_STATE   SOURCE_TEXT   
+-------------------------  --------------------  ----------------------------------------------------------------------------  --------------- ------------------------------------------------------------------------------------------------------------------------------                                               
+013-02-22 16:29:30.430629                  1772  MXID11000030466212228308797313024000000000106U6553500_5_SQL_DATASRC_Q1        FETCH           "select translate(nid.obj_name using UCS2TOUTF8), nid.obj_id, asd.default_type, asd.automation, dsn.max_srvr_cnt, dsn.avail_sr                                               
+013-02-22 16:29:30.430629                  1772  MXID11000030466212228308797313024000000000106U6553500_7_SQL_DATASRC_Q7        FETCH           "select variable_sequence, variable_type, translate(variable_value using UCS2TOUTF8) from HP_SYSTEM_CATALOG.MXCS_SCHEMA.NAME2I                                               
+013-02-22 16:29:30.430629                  1772  MXID11000030466212228308797313024000000000106U6553500_6_SQL_ENV_Q6            FETCH           "select cast(COUNT(*) as int) from HP_SYSTEM_CATALOG.MXCS_SCHEMA.ENVIRONMENTVALUES where env_id = cast(? as integer) FOR READ
+ 
+
+ACTIVE QUERIES
+================
+
+This statement lists queries which either have finished executing within the
+given number of seconds or still being executed. The negative number in 
+LAST_ACTIVITY_SECS means that the query has completed that many seconds ago. 
+
```

Comment:
If the query is still executing, LAST_ACTIVITY_SECS will be the number of seconds(a positive number) the query has been running till then. The queries are listed in the descending order of LAST_ACTIVITY_SECS to help the customer to focus on these queries first